docs/explanations/using_pro_offline.rst

```diff
@@ -1,61 +1,77 @@
+  provides the security support and other benefits of Pro even without internet
+  access.
+  
+* **Firewalled deployment**
+  Although not *truly* airgapped, this refers to systems where network
+  connectivity is heavily limited. On-prem setups tend to fall under this
+  method, for example, with Livepatch On-prem -- where updates are fetched from
+  the Canonical servers to a local machine. This provides more control over
+  when, how, and which updates are deployed. 
 
 How do I sync Pro packages in an airgapped system?
 ==================================================
 
-This is an advanced use-case, and our `Customer Support teams`_ can help you
-implement this in the best way for your specific environment. Note that this
-requires a paid subscription to Ubuntu Pro.
+No matter whether you are running a truly airgapped (offline) system, or one
+that is heavily firewalled, running Ubuntu Pro airgapped is an advanced
+use-case. In both cases, you will need to contact our `Customer Support teams`_
+who can help you implement air gapping in the best way for your specific
+environment.
 
-What tools do I need?
-=====================
+Note that this requires a paid subscription to Ubuntu Pro.
 
-Updating Ubuntu systems in an offline environment requires mirroring Ubuntu
-repositories. Since Ubuntu packages can be consumed via deb packages and/or
-snaps, the tool needed to mirror Ubuntu repositories depends on where you
-source your packages from.
+What tools will I need?
+=======================
+
+Updating Ubuntu systems in a truly offline environment requires mirroring
+Ubuntu repositories. Since Ubuntu packages can be consumed via deb packages
+and/or snaps, the tool needed to mirror Ubuntu repositories depends on where
+you source your packages from.
 
 Mirror deb repositories with Landscape
 --------------------------------------
 
 Landscape is a management tool that can host a mirror of Ubuntu deb
-repositories. 
+repositories. It can be used in both kinds of airgapped environments.
 
-In future release cycles, Landscape will also include features specific to
-airgapped use-cases. You can keep up to date with these developments in the
-`Landscape Beta Discourse`_.
+In future release cycles, Landscape will also include specific features to
+support airgapped use-cases. You can keep up to date with these developments in
+the `Landscape Beta Discourse`_.
 
 Mirror snaps with Snap store proxy
 ----------------------------------
 
-The `Snap-store-proxy`_ is the equivalent tool for hosting the snap packages that
+The Snap store proxy is the equivalent tool for hosting the snap packages that
 your organisation is consuming.
+
+`Snap-store-proxy`_ is a snap that provides an edge proxy to the Snap store, so
+that after your device has been registered with the proxy, all communications
+with the Snap store will be sent via the proxy. This is part of the "firewalled"
+deployment.
```

Comment:
Snap store proxy does have a true airgapped mode that we can link to https://docs.ubuntu.com/snap-store-proxy/en/airgap

Reply:
Nice find! Added a reference to it